rook/main.py

```diff
@@ -202,20 +232,28 @@ def process_result(index, _input_data, output_data):
 
   test_re = re.compile(args.test_re_raw)
 
-  #XXX fixme to find a better way to the tests directory
-
   this_dir = os.path.abspath(os.path.dirname(__file__))
   up_one_dir = os.path.dirname(this_dir)
-  base_test_dir = os.path.join(up_one_dir, "tests")
+  if args.test_dir is None:
+    #XXX fixme to find a better way to the tests directory
```

Comment:
I assume this is still something to do in the future?

Reply:
Yes, this probably should be done when we split rook to a separate repository.